dashboard/README.md

```
@@ -0,0 +1,228 @@
+
+### Deploy Manual
+
+```bash
+npm run build
+```
+
+O build estará em `.next/`. Você pode hospedar em qualquer serviço que suporte Next.js.
+
+## 📁 Estrutura do Projeto
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)